clusterloader/config/rctest.yaml

```diff
@@ -0,0 +1,38 @@
+provider: local
```

Comment:
I see. So that should be passed from the script running test, not being part of config which should be provider-independent.